includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -240,6 +167,92 @@ public function __construct(
 			],
 		];
 
+		// Feature flag.
+		// TODO: Remove this check and inject contents of `$payment_request_fields` into `$this->form_fields`
+		// after `saved_cards` when Apple Pay gets released.
+		if ( version_compare( get_option( 'woocommerce_woocommerce_payments_version', WCPAY_VERSION_NUMBER ), '3.0.0', '>=' ) ) {
```

Comment:
I don't think version compare is a good approach to feature flag here. Custom option is better,e.g. `_wcpay_feature_payment_request`.